MAUI/ImageEditor/text.md

```diff
@@ -0,0 +1,146 @@
+        TextAlignment = TextAlignment.Start, 
+        TextStyle = new ImageEditorTextStyle() 
+        { 
+            FontSize = 14,
+            TextColor = Colors.Black, 
+            FontFamily = "Arial", 
+            FontAttributes = FontAttributes.Italic 
+        } 
+    }); 
+
+{% endhighlight %}
+{% endtabs %}
+
+## Delete the text
+
+Delete the selected text using either the toolbar or the [`DeleteAnnotation`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_DeleteAnnotation) method.
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+    imageEditor.DeleteAnnotation();
+
+{% endhighlight %}
+{% endtabs %}
+
+## Clear all texts
+
+Remove all the text annotations using the [`ClearAnnotations`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ClearAnnotations) method.
+
+N> This removes shape and pen annotations as well.
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+    imageEditor.ClearAnnotations();
+
+{% endhighlight %}
+{% endtabs %}
+
+## AnnotationSelected event
+
+This [`AnnotationSelected`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_AnnotationSelected) event occurs when the annotation is selected.
+
+N> This is common for Shape and Text annotations.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+    <imageEditor:SfImageEditor Source="image.png" AnnotationSelected = "OnAnnotationSelected" />
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="C#" %}
+
+    private void OnAnnotationSelected(object sender, AnnotationSelectedEventArgs e)
+    {
+        if (e.AnnotationSettings is ImageEditorTextSettings textSettings)
+        {
+            textSettings.RotationAngle = 90;
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add text on initial loading
+
+Add a text on image loading using the [`Imageloaded`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ImageLoaded) event.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+    <imageEditor:SfImageEditor x:Name="imageEditor" Source="image.png" ImageLoaded = "OnImageLoaded" />
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="C#" %}
+
+    private void OnImageLoaded(object sender, EventArgs e)
+    {
+        this.imageEditor.AddText("Good morning");
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add text with manual bounds
+
+The text can be added with user-defined view bounds. The [`Bounds`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_Bounds) are treated as ratio values of image width and height, so you have to specify bound's rectangle values in the range of 0 to 1
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
```

Comment:
is this method need to mention in OnImageLoaded event? add examples

Reply:
Added examples using OnImageLoaded event.